src/crypto/olmlib.js

```diff
@@ -407,6 +407,15 @@ export function encodeBase64(uint8Array) {
     return Buffer.from(uint8Array).toString("base64");
 }
 
+/**
+ * Encode a typed array of uint8 as unpadded base64.
+ * @param {Uint8Array} uint8Array The data to encode.
+ * @return {string} The unpadded base64.
+ */
+export function encodeUnpaddedBase64(uint8Array) {
+    return encodeBase64(uint8Array).replace(/=$/g, '');
```

Comment:
```suggestion
    return encodeBase64(uint8Array).replace(/=+$/g, '');
```

or something, since base64 can have more than one `=` at the end

Reply:
indeed
![image](https://user-images.githubusercontent.com/1190097/75467734-53b3d300-5949-11ea-8a32-6c472c1f20b6.png)